master/master.cfg

```diff
@@ -324,17 +324,22 @@ def get_targets(os, llvm):
     targets.extend([('test_apps', 'host'),
                     ('test_tutorial', 'host')])
 
-    if not os.startswith('mac'):
-      # The linux and mingw build-bot has an nvidia card
+    if os.startswith('linux-64-gcc53') or os.startswith('mingw'):
```

Comment:
This if tree is getting confusing:

    if not mingw {
        stuff
    }
    if linux-32 or mac-32 {
        stuff
    }
    if linux-64 or mingw-64 or mac-64 {
        if not mingw {
            stuff
        }
        stuff
        if linux-64 or mingw {
            stuff
        }
        if linux-64 {
            stuff
        }
        if mac {
            stuff
        }
    }
 
Maybe flatten in one level? I.e. remove the extended cpu/gpu tests 'if' and just make everything top-level?

Reply:
Done, PTAL